src/main/kotlin/com/hedvig/rapio/insurancecompanies/dto/InsuranceCompanyDto.kt

```diff
@@ -0,0 +1,11 @@
+package com.hedvig.rapio.insurancecompanies.dto
+
+import com.hedvig.rapio.externalservices.underwriter.transport.InsuranceCompanyDto as UnderwriterInsuranceCompanyDto
+
+data class InsuranceCompanyDto(
+    val id: String,
+    val displayName: String,
+    val switchable: Boolean
+)
+
+fun UnderwriterInsuranceCompanyDto.toDto(): InsuranceCompanyDto = InsuranceCompanyDto(id, displayName, switchable)
```

Comment:
Is there any benefit to having it as an extension in this scenario? Maybe readability?
```suggestion
data class InsuranceCompanyDto(
    val id: String,
    val displayName: String,
    val switchable: Boolean
) {
    fun toDto() = InsuranceCompanyDto(id, displayName, switchable)
}
```

Reply:
Different classes here too. Maybe the fact that you misread it is an indication that it should be changed though 😄